### ISSUE TITLE
Wiki: final steps for Pi

### ISSUE THREAD

**Issue by Cheese-oOo** [meduzapat/LEDSpicerUI#27]
After command: sudo make install
Add:

sudo mkdir /usr/local/share/ledspicerui
sudo cp -a ./data/. /usr/local/share/ledspicerui

**Comment by meduzapat** [comment]
Good catch. Starting with version 0.7.0, LEDSpicer introduced projects, which changed how files are discovered and managed. A legacy backward-compatibility mode is still available for users who do not want to use projects.

The UI now does the following:

1. Checks if ledspicerd is available
2. Queries the configured paths
3. Uses those paths automatically
4. Allows paths to be changed
5. Uses projects located inside the projects directory
6. Supports both local and deploy modes (where files are copied to a different system)

**Comment by meduzapat** [comment]
I will update the wiki with the newer version
